app/uk/gov/hmrc/trackingconsentfrontend/assets/src/ui/renderSettingsSaveConfirmationMessage.ts

```diff
@@ -8,12 +8,13 @@ import callIfNotNull from '../common/callIfNotNull';
 import scrollToTop from '../common/scrollToTop';
 import getReferrer from '../common/getReferrer';
 // @ts-ignore
```

Comment:
Do we still need the @ts-ignore?

Reply:
Indeed, should be able to remove that now.